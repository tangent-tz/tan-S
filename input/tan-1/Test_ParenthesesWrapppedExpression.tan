main {
    const x := 4 * (2 + (-3 * ((2 + 7) * 6)));
    print x;

    # Expected output: -640
}


