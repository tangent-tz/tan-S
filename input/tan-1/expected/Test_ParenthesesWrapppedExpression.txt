-640
111.200000
0
